anyway/widgets/suburban_widgets/motorcycle_accidents_vs_all_accidents_widget.py

```diff
@@ -163,5 +166,12 @@ def localize_items(request_params: RequestParams, items: Dict) -> Dict:
         }
         return items
 
+    # noinspection PyUnboundLocalVariable
+    def is_included(self) -> bool:
+        motor_accidents_in_road = self.items[0]["series"][0]["value"]
+        motor_accidents_in_all_roads = self.items[1]["series"][0]["value"]
+        # At least 3 severe motorcycle accidents in road and more than twice as much of motor accidents in other roads
+        return motor_accidents_in_road >= 3 and motor_accidents_in_road >= 2*motor_accidents_in_all_roads

```

Comment:
In items we calculate percentages rather than accidents count. Hence, the filtering logic is not correct.
Perhaps, we can save the relevant accidents count data (when performing calculation in `motorcycle_accidents_vs_all_accidents`) in an additional instance variable, for example items_metadata.
@BarVolunteering  @ziv17 what do you think?

Reply:
(This means while in `motorcycle_accidents_vs_all_accidents ` saving the following variables:
`counter_road_motorcycle` (and in  `is_included` using counter_road_motorcycle > 3)
And saving the result of `counter_other_motorcycle + counter_road_motorcycle` to get motor accidents in all roads for the second comparison

Reply:
(This means while in `motorcycle_accidents_vs_all_accidents ` saving the following variables:
`counter_road_motorcycle` (and in  `is_included` using counter_road_motorcycle > 3)
And saving the result of `counter_other_motorcycle + counter_road_motorcycle` to get motor accidents in all roads for the second comparison

Reply:
Yes, we are returning percentages, setting local variables with the data sound like a good solution

Reply:
I also agree. 
One point is the percentage Vs. actual numbers
Another point is that relying on the order of the fields results in less readable code, So it will be easier to maintain when using intermediate instance variables for the values tested in `is_included`.

Reply:
@shaniwein is this fixed? (I think not since in items we still have the percentages, but maybe I missed something?)

Reply:
It is fixed now.